src/celeritas/ext/ScopedRootErrorHandler.cc

```diff
@@ -44,7 +44,8 @@ void RootErrorHandler(Int_t rootlevel,
     {
         level = LogLevel::error;
         g_has_root_errored_ = true;
-        if (std::strcmp(location, "TCling::LoadPCM") == 0)
+        if (std::strcmp(location, "TCling::LoadPCM") == 0
+            || std::strcmp(location, "TProtoClass::FindDataMember") == 0)
```

Comment:
Maybe it would be good to use `string_view` to improve readability?  e.g.:
```suggestion
        using namespace std::literals::string_view_literals;
        if (location == "TCling::LoadPCM"sv || location == "TProtoClass::FindDataMember"sv)
```
and if the list continues to grow, you could make a `static string_view const fatal_location[] = {};` and use `find(begin(fatal_location), end(fatal_location), string_view{location}) != end(fatal_location))`.

Also now that the `if` statement is longer than the body it's definitely time to add some braces around the line below.